connect/Env.hx

```diff
@@ -55,6 +56,7 @@ class Env extends Base {
     private static var generalApi: GeneralApi;
     private static var marketplaceApi: MarketplaceApi;
     private static var subscriptionsApi: SubscriptionsApi;
+    private static inline var rootLogger: String = "root";
```

Comment:
Constants should be named in upper snake case (`ROOT_LOGGER`) to be consistent with the rest of the SDK. Also, please put it before variable declarations, separated from those with a blank line.